mkdocs.yml

```diff
@@ -90,6 +90,14 @@ plugins:
   macros:
       include_yaml:
 #         - 'variables.yml' # Use in markdown as '{{ VAR }}'
+  with-pdf:
+    output_path: '_pdf/Percona Platform.pdf'
+    cover_title: 'Percona Platform documentation'
+    cover_subtitle: latest update
+    author: "Percona Technical Documentation Team"
+    cover_logo: docs/_images/Percona_Logo_Color.png
+    custom_template_path: _resources/templates
```

Comment:
Seems like `_resources` should be `resources`. I'm not even sure it has any effect, since the PDF got ultimately built :)